README.md

```diff
@@ -204,8 +197,21 @@ _* Mono JIT compiler (tested with version 4.6.0)_
   nunit-console src/Tests/bin/Release/Tests.dll --config=Release
   ```
 
+## Publish to the Nearmap NuGet Repository
+
+Build it first. See above.
 
+### on Linux, OS X
 
+1. Build the NuGet package
+  ```
+  mono lib/NuGet.exe pack src/StatsdClient/StatsdClient.nuspec -Version '<version>' -BasePath <path>/src/StatsdClient -IncludeReferencedProjects
+  ```
+
+2. Publish the NuGet package
+  ```
+  curl -v --fail -s -S -X PUT -H "X-NuGet-ApiKey:<secretkey>" -F package=@DogStatsD-CSharp-Client.<version>.nupkg http://nuget.development.nearmap.com/api/packages
```

Comment:
could probably use `NuGet.exe push` - our makefile doesn't only because I found it unreliable for the Framework, which pushes many packages consecutively - for single packages it was ok.